assets/src/components/dup/README.md

```diff
@@ -9,6 +9,7 @@
 - Create priv/static/dup-app.html if it doesn’t already exist. Copy paste contents as directed below.
```

Comment:
while you're in here, do you think it'd be possible to move the dup-app.html code sample to underneath this bullet? it's kind of a weird flow of steps to hop down beyond the end of the instructions then back up to continue.